checkr/src/models/candidate-report.ts

```diff
@@ -0,0 +1,53 @@
+import { Table, Column, Model, ForeignKey, BelongsTo, DataType } from 'sequelize-typescript';
+import { Candidate } from './candidate';
+
+enum Status {
```

Comment:
do research on enums 

it is not much useful, instead use like this
```
export const CandidateStatus = {
  /**as we put candidate information in db */
  sourced: 'sourced',
  /** if we initiated the court search */
  court_searching: 'court_searching',
  /** court_search_complete */
  court_search_complete: 'court_search_complete',
  /** request_for_documents:
   * This status is also related with scheduled status
   * this is equivalent to pre-adverse initiated.
   */
  request_for_documents: 'request_for_documents',
  /** In_Review – after user submits requested documents, from this it can again go to request for document */
  in_review: 'in_review',
  /** Engaged – recruiter thinks it is better to engage with candidate, regardless of court search result */
  engaged: 'engaged',
  /** Rejected – recruiter rejects candidate on account of violations */
  rejected: 'rejected',
  /** Canceled – recruiter cancels candidate flow */
  canceled: 'canceled',
} as const;

export type CandidateStatusType =
  (typeof CandidateStatus)[keyof typeof CandidateStatus];

```

Reply:
okay. will look into it.